packages/grafana-ui/src/components/Select/resetSelectStyles.ts

```diff
@@ -21,6 +21,7 @@ export default function resetSelectStyles(theme: GrafanaTheme2) {
         padding: 0,
         // Set an explicit z-index here to ensure this element always overlays the singleValue
         zIndex: 1,
+        overflow: 'hidden',
```

Comment:
Not super fond of this boing i `resetSelectStyles`

If we stick it in `useCustomSelectStyles` we would have to merge it with the input from `resetSelectStyles`